api_tests/lib/config.ts

```diff
@@ -11,7 +11,7 @@ export interface CndConfigFile {
 }
 
 export interface HttpApi {
-    socket: { address: string; port: number };
```

Comment:
Don't forget to write a changelog update for this breaking change :)

Reply:
Added, thanks.